threenums.c

```diff
@@ -0,0 +1,40 @@
+/* Determines whether 3 numbers are equal, not equal, which of the three numbers is largest (if that is the case) and the sum of the three numbers.     - Amanda Cuevas */
+#include <stdio.h>
```

Comment:
Really nice job with comments, but your code doesn't properly handle the case where 2 numbers are equal and also the largest number.